src/test/java/org/folio/des/scheduling/quartz/EdifactExportJobSchedulerTest.java

```diff
@@ -160,6 +160,28 @@ void testJobRemovedForRemovedConfiguration() throws SchedulerException {
       () -> assertEquals(0, scheduler.getJobKeys(GroupMatcher.anyJobGroup()).size()));
   }
 
+  @Test
+  void testDeleteJobGroup() throws SchedulerException {
+    ExportConfig config = getExportConfig();
+    edifactExportJobScheduler.scheduleExportJob(config);
+
+    edifactExportJobScheduler.deleteJobGroup(TENANT);
+
+    var jobKeys = scheduler.getJobKeys(GroupMatcher.anyJobGroup());
+    assertEquals(0, jobKeys.size());
+  }
+
+  @Test
+  void testNotDeleteIfJobDontExists() throws SchedulerException {
+    ExportConfig config = getExportConfig();
+    edifactExportJobScheduler.scheduleExportJob(config);
+
+    edifactExportJobScheduler.deleteJobGroup("");
+
+    var jobKeys = scheduler.getJobKeys(GroupMatcher.anyJobGroup());
+    assertEquals(1, jobKeys.size());
+  }
+
```

Comment:
One more test case to add: add jobs for several tenants and check that jobs are removed only for expected one

Reply:
I will add